tracing/init.go

```diff
@@ -42,8 +42,8 @@ func InitTracer(serviceName, instanceName string) (trace.Tracer, func(), error)
 // TracerProvider will also use a Resource configured with all the information
 // about the application.
 func newTracerProvider(serviceName, instanceName string) (*tracesdk.TracerProvider, error) {
```

Comment:
I think the InitTracer function above should be changed to return `TracerProvider`, not a tracer.